src/main/webapp/deletefile.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="deletefile.css">
+    <!-- Bootsrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" 
+    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <script src="script.js"></script>
+    <!-- Dependencies for Bootstrap, jQuery, and Popper.js. -->
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" 
+    integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" 
+    integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" 
+    integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <h1>Billing Configuration Sync Tool</h1>
+    <h2>Delete Configuration</h2>
+    <div id="links">
+      <a href="readfile.html">Read Configuration</a>
+      <a href="editfile.html">Edit Configuration</a>
+      <a href="createfile.html">Create Configuration</a>
+    </div><br>
+    <div id="delete-search">
+      <form onsubmit="openModal()" id="deleteForm">
+        <span>Customer ID:</span>
+        <input type="text"><br>
+        <span>Account ID:</span>
+        <input type="text"><br>
+        <button type="button" class="btn-primary" data-toggle="modal"
+        data-target="#deleteModal">Submit</button>
+      </form>
+    </div>
+    <!-- TODO: Add modal confirmation. -->
+    <div class="modal fade" tabindex="-1" role="dialog" id="deleteModal">
```

Comment:
"modal fade" -> "modal-fade"?

Reply:
+1, modal fade to modal-fade will make this consistent with the class fields below.

Reply:
Thank you for the comment clarifying this.